pages/common/pnpm.md

```diff
@@ -0,0 +1,40 @@
+# pnpm
+
+> Fast, disk space efficient package manager for Javascript and Node.js.
+> Manage Node.js projects and their module dependencies.
+> More information: <https://pnpm.io//>.
+
+- Interactively create a `package.json` file:
+
+`pnpm init`
+
+- Download all the packages listed as dependencies in package.json:
+
+`pnpm install`
+
+- Download a specific version of a package and add it to the list of dependencies in `package.json`:
+
+`pnpm install {{module_name}}@{{version}}`
+
+- Download a package and add it to the list of dev dependencies in `package.json`:
+
+`pnpm install {{module_name}} --save-dev`
+
+`pnpm install -D {{module_name}}`
```

Comment:
Do you have to execute both of the commands? If so, we should write
 `pnpm install {{module_name}} --save-dev && pnpm install -D {{module_name}}`

Also, we use long arguments where possible:
```suggestion
`pnpm install {{module_name}} --save-dev`

`pnpm install --dev {{module_name}}`
```

Reply:
Nah it was just 2 types on how you can install dev dependencies. Either by doing `--dev` or `--save-dev` or by simply `-D`

Reply:
I think that most clients don't support having two command lines. We have to remove one of the commands.

Reply:
Yeah in here, I removed the the `--save-dev` one and only kept `-D` one, should i change it to `--dev`? but i think `-D`is more than enough

Reply:
We generally prefer the long variants where possible, so users understand the command instead of just remembering it.